vagrant/hieradata/common.yaml

```diff
@@ -0,0 +1,29 @@
+# Infrastructure
+cmf::infrastructure::packages::installs:
+  - vim
+  - git
+  - wget
+  - bash-completion
+  - ruby-dev
+  - software-properties-common
+  - python-software-properties
+
+# Application
+cmf::application::php::version: "5.6" # Available are "5.4", "5.5", "5.6" and "7.0"
+
+# Storage
+cmf::infrastructure::storage::mysql: true
+cmf::infrastructure::storage::jack: false
+
+# MySQL
+cmf::infrastructure::phpcr::mysql_wrapper::root_password: root
+cmf::infrastructure::phpcr::mysql_wrapper::databases:
+  cmf-database:
```

Comment:
we don't need multiple databases as we just need one for the jackalope-doctrine-dbal. Multiple databases would confuse puppet since it doesn't now which one it should use for jackalope